### DISCUSSION TITLE
2024-03-24 오프라인 회의

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [Dev-On-Play/spring-project#12] [오프라인 회의록]
# 서비스 명 결정
모각코 스케쥴러(MOS)

- BE : API [명세서](https://github.com/Dev-On-Play/spring-project/wiki/%5BBE%5D-%EA%B5%AC%ED%98%84%ED%95%A0-API-%EB%AA%85%EC%84%B8) 대로 티켓을 만들고 해당 번호로 브랜치를 생성해서 작업하기(Git-flow)
- FE : 화면별 개발을 진행한다.

- 당분간은 오프라인 모임 보다는 온라인 모임으로 진행(다음은 04월07일 쯤 진행, 시간은 추후 투표)
- 로그인 페이지는 먼저 구글 연동으로 시작(추후 다양하게 연동)
- ELK 등의 추가 기능은 BE팀의 초기 작업이 끝나고 난 후 기간이 있을때 추가 작업
-  ~~대댓글 1개 뎁스~~
    - 메인 기능 완료 후 추가 개발할 경우에, 1뎁스만
- 모각코 상세(생성자)는 수정은 가능하지만, 1명이라도 참여하면 수정 불가능
- ~~페이지별 개발~~
- 모각코 서브 타이틀(메인페이지에서 나오는)이 필요할것 같다.
- 모각코 생성시 에디터를 고도화 한다면 어떤걸 쓸지 정해야 한다.(기본적으로 textarea도 상관없음)
- API 명서세에 나온 기능별 개발

**Comment by rave189** [comment]
대댓글은 중요도가 낮아 나중에 개발하기로 했던 것 같습니다.
페이지별 개발이 메인 페이지의 모각코 리스트를 페이지네이션으로 개발하는 걸 의미하는 걸까요?

**Reply by hdwhdwhdw** [reply]
대댓글은 "메인 기능 완료 후 추가 개발할 경우에, 1뎁스만" 으로 수정하겠습니다!

**Reply by hdwhdwhdw** [reply]
"페이지별 개발" 은 "FE : 화면별 개발을 진행한다." 이거랑 중복되는 내용이네요.
삭제하도록 하겠습니다.

**Comment by hdwhdwhdw** [comment]
API 명세는 [여기](https://github.com/Dev-On-Play/spring-project/wiki/%5BBE%5D-%EA%B5%AC%ED%98%84%ED%95%A0-API-%EB%AA%85%EC%84%B8)로 옮겼으니, 확인 부탁 드립니다!
